Cleanup in repo (#724)
Altinn/app-frontend-react: README.md

3. Follow our documentation on [how to use the local app-frontend](https://docs.altinn.studio/app/testing/local/debug/#using-other-frontend-versions) when running locally

## Automated tests

### End to end tests

End to end tests are using Cypress, see [test readme for how to run these](./cypress/README.md).

### Unit tests

Unit tests are written using Jest and React testing library

1. Execute `yarn --immutable`. This step is only nescessary if you have not already done it, or when package.json changes.
2. Execute `yarn run test`.

## Contributing

Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.

### Code style

We use [eslint](https://eslint.org/) and [prettier](https://prettier.io/), and automatically set up git hooks to enforce
these on commits. To avoid confusion, it is recommended to set this up your IDE:

<details>
<summary>Visual Studio Code</summary>
Install the [eslint extension from the marketplace](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).

> </details>

Reviewer: Maybe also mention the prettier extension?
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

There is also an `extensions.json` file that suggests recommended extensions for the workspace, maybe this should be updated as well after moving since c-sharp is not really useful in this repo.

Reviewer: Updated this file here: https://github.com/Altinn/app-frontend-react/pull/727
The readme could also say to simply install the recommended extensions.

Author: Eslint is set up to run prettier automatically, so I don't think you need the prettier extension? That will just do the job twice, and possibly cause problems for eslint (as we have set up eslint to override stuff from prettier if we want to).

Author: I don't really use vscode myself, but if the recommended extensions pop up when opening the project, then yes, I guess we could just write that? :shrug: Could you formulate something meaningful for vscode users (feel free to commit to this branch)?

Reviewer: It is possible that the prettier extension is not needed, actually not sure how it works

> <details>
> <summary>Visual Studio Code</summary>
> Install the [eslint extension from the marketplace](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).
> </details>

Reviewer: ```suggestion
<details>
<summary>Visual Studio Code</summary>
Install the extensions recommended by the workspace, at least [eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).
</details>
```
